po/it.po

```diff
@@ -440,68 +442,59 @@ msgid "Quit"
 msgstr "Esci"
 
 #: src/shortcuts_dialog.ui:15
-#, fuzzy
 msgid "Change Drawing Mode"
-msgstr "Modalità disegno"
+msgstr "Cambia modalità di disegno"
 
 #: src/shortcuts_dialog.ui:21
-#, fuzzy
 msgid "Change Drawing Area Margin"
-msgstr "Margine dell'area di disegno"
+msgstr "Modifica margine area di disegno"
 
 #: src/shortcuts_dialog.ui:27
-#, fuzzy
 msgid "Change Offset Between Items"
-msgstr "Scostamento tra gli elementi"
+msgstr "Modifica scostamento tra gli elementi"
 
 #: src/shortcuts_dialog.ui:33
 msgid "Change Roundness of Items"
-msgstr ""
+msgstr "Cambia rotondità degli oggetti"
 
 #: src/shortcuts_dialog.ui:39
-#, fuzzy
 msgid "Toggle Sharp Corners"
-msgstr "Angoli acuti"
+msgstr "Attiva/disattiva angoli acuti"
 
 #: src/shortcuts_dialog.ui:45
-#, fuzzy
 msgid "Toggle Window Controls"
-msgstr "Dimensione finestra"
+msgstr "Attiva/disattiva i controlli della finestra"
 
 #: src/shortcuts_dialog.ui:51
 msgid "Toggle Autohide Headerbar"
-msgstr ""
+msgstr "Attiva/disattiva automaticamente la barra di intestazione"
 
 #: src/shortcuts_dialog.ui:62
-#, fuzzy
 msgid "Change Number of Bars"
-msgstr "Numero di barre"
+msgstr "Cambia numero di barre"
 
 #: src/shortcuts_dialog.ui:68
-#, fuzzy
 msgid "Toggle Channels"
-msgstr "Canali"
+msgstr "Abilita/disabilita canali"
 
 #: src/shortcuts_dialog.ui:74
 msgid "Toggle Reverse Order"
-msgstr ""
+msgstr "Attiva/disattiva ordine inverso"
 
 #: src/shortcuts_dialog.ui:85
-#, fuzzy
-msgid "Toggle Widgets Style"
-msgstr "Stile widget"
+msgid "Change Widgets Style"
+msgstr "Cambia lo stile widget"
```

Comment:
Please translate "**Toggle** Widgets Style" here, this is recent change, sorry for inconvenience

Reply:
Attiva/disattiva lo stile dei widget

Reply:
Idk how it should be in Italian, the thing is you have old `msgid` there ("Change...", but I recently changed it to "Toggle...")